drafts/maui-test_pt1.md

```diff
@@ -0,0 +1,325 @@
+---
+title: How to test a .NET MAUI App Part 1
+domain: software-engineering-corner.zuehlke.com
+tags: dotnet,mvvm,xaml,net-maui,csharp,xunit,testing,moq
+cover: https://cdn.hashnode.com/res/hashnode/image/upload/v1715274155168/4q4lrfJcY.png?auto=format
```

Comment:
Tiny thing: having the title of the article over the cover image seems a tad unnecessary and doesn't quite fit with the other articles so far. But I get the idea of making sure the two parts are visually connected. Maybe using the same image and "part 1" & "part 2" would be enough? Or at least have "part X" on a new line.

Reply:
I totally agree with your reasoning. I adopted your first suggestion.